scripts/remove-purposes.js

```diff
@@ -46,7 +46,7 @@ const main = async () => {
       shouldUpdate = true
     }
     if (args.remove_recurring) {
-      delete doc.aggregation.recurring
+      delete doc.aggregation?.recurring
```

Comment:
Why don't we do the same here https://github.com/cozy/coachCO2/blob/2c95f6bed3557f301bcd037dc3285d669281772f/scripts/remove-purposes.js#L39 ? 

Is it wanted to pass to this method without any aggregation? Is it the right place to fix the real issue?

Reply:
Honestly, I just quickly fixed it after I got an error by running this script, that I forgot the existence :)
But you're right, we should simply only query docs with aggregation here